packages/blockchain/src/consensus/ethash.ts

```diff
@@ -7,15 +8,18 @@ import { Consensus, ConsensusOptions } from './interface'
  * This class encapsulates Ethash-related consensus functionality when used with the Blockchain class.
  */
 export class EthashConsensus implements Consensus {
-  blockchain: Blockchain
-  _ethash: Ethash
+  blockchain: Blockchain | undefined
```

Comment:
Same here regarding the `undefined`

Reply:
See my earlier comment as the same applies. In practice, this Blockchain property should never be undefined because the `setup` method is called right after the blockchain is instantiated.